funding/manager.go

```diff
@@ -3240,7 +3217,7 @@ func (f *Manager) annAfterSixConfs(completeChan *channeldb.OpenChannel,
 		// For private channels we do not announce the channel policy
 		// to the network but still need to delete them from the
 		// database.
-		err = f.deleteInitialFwdingPolicy(chanID)
+		err = f.deleteInitialForwardingPolicy(chanID)
```

Comment:
Maybe we can keep it? Remember someone asked whether it's possible to change a private channel into a public one. Anyway just a thought non-blocking.

Reply:
I don't think we should. this record is only stored to "survive" the time from the channel funding being initiated by the user/RPC until it's announced to the graph. And as far as I understand the values don't really have any meaning for private channels. So if we keep the data, we'd also need to make sure to clean them up outside of the funding manager.

It's also worth noting here that the values we store here can be overwritten any time after the channel is confirmed by using the `UpdateChannelPolicy`. So if we ever add an RPC to make a private channel public, I would expect us to just add the same policy parameters to that RPC so the user can specify them at the time of making the channel public (since the desired fee rates might have changed since the channel's creation).